### ISSUE TITLE
Combining X Below

### ISSUE THREAD

**Issue by nardog** [clld/phoible#18]
Non-IPA symbol Combining X Below (U+0353) is used in some consonants and vowels. It seems as though it stands for frication, but this is neither sanctioned by the IPA nor standard or common. I suggest it be replaced with whatever appropriate according to the description of the sources.

**Comment by drammock** [comment]
You are correct, it is not part of the official IPA.  In fact there are several symbols used in the PHOIBLE dataset that are not part of the official IPA. Most of them are documented [here](http://phoible.github.io/conventions/), though we just pushed out a major release and the conventions webpage has not been updated since then (but will be soon).

With regard to your suggestion that we replace U+0353 with "whatever is appropriate according to the description of the sources" --- well, the reason we use that diacritic is because we believe the official set of IPA symbols and diacritics is not sufficient to adequately capture the variety that is present across the roughly 3000 inventories in PHOIBLE.  If there are specific language descriptions where you think our use of U+0353 is unwarranted given the description of the sounds in the cited reference, we welcome you to open issues or pull requests on [the development repository](https://github.com/phoible/dev) to call our attention to it.

**Comment by nardog** [comment]
Drammock, thank you for your reply and the link to the documentation. I would argue that it could be replaced by symbols officially sanctioned by the IPA in many cases, but that might entail some amount of subjective interpretation of the literature, which I understand is best avoided.

So long as the value of each symbol is defined, I see no problem using non-IPA symbols. But I don't see a link to or a copy of the documentation on the online version (phoible.org). I think there definitely should be one, is there reason why it's not there?

**Comment by drammock** [comment]
Your initial message made me also realize that the documentation is not prominent from the main website. @xrotwang is there any difficulty in adding a couple static HTML pages to the phoible.org domain?

**Comment by xrotwang** [comment]
Nope,  that's simple enough. Let me know which ones you want and linked
from where.

Daniel McCloy <notifications@github.com> schrieb am So., 7. Apr. 2019,
09:55:

> Your initial message made me also realize that the documentation is not
> prominent from the main website. @xrotwang is there any difficulty in
> adding a couple static HTML pages to the phoible.org domain?
>

**Comment by drammock** [comment]
Great. Once @bambooforest and I have updated the docs I'll ping you about where to put them.
